src/Common/Key_Value_Cache/Key_Value_Cache_Table.php

```diff
@@ -0,0 +1,215 @@
+<?php
+/**
+ * The API of the key-value cache.
+ *
+ * @since TBD
+ *
+ * @package Common\Key_Value_Cache;
+ */
+
+namespace TEC\Common\Key_Value_Cache;
+
+use Exception;
+use TEC\Common\Key_Value_Cache\Table\Schema;
+use TEC\Common\StellarWP\DB\DB;
+
+/**
+ * Class Key_Value_Cache_Table.
+ *
+ * @since TBD
+ *
+ * @package Common\Key_Value_Cache;
+ */
+class Key_Value_Cache_Table implements Key_Value_Cache_Interface {
+	use Key_Value_Cache_Methods_Trait;
+
+	/**
+	 * Whether a key exists in the cache.
+	 *
+	 * This method tells nothing about the nature of the value, it only checks whether the key exists in the cache.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $key The key to check.
+	 *
+	 * @return bool Whether the key exists in the cache.
+	 */
+	public function has( string $key ): bool {
+		$current_time = time();
+
+		try {
+			$exists = DB::get_var(
+				DB::prepare(
+					'SELECT COUNT(*) FROM %i WHERE `cache_key` = %s AND (`expiration` = 0 OR `expiration` > %d)',
```

Comment:
It seems like we dont support excpiration 0 anymore (5 mins min), so maybe we should amend the queries ?